renovate-config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "description": "Default configuration for repositories in the Mezzio organisation",
+    "extends": [
+        "local>laminas/.github:renovate-config"
```

Comment:
Is this indirection fine? Overall OK from me, just making sure we don't create too many hops for renovate to follow :D

Reply:
No no, it's just the 1 hop from laminas to here and to Laminas API Tools. It let's us share that config within these orgs.